### DISCUSSION TITLE
is DNSSEC supported?

### DISCUSSION THREAD

**Discussion by rekomat** [vercel/community#331] [Help] [answered]
Hi there — Couldn't figure out whether `DNSSEC` is supported… is it?

**Comment by rekomat** [comment, marked as answer]
Nope … 

![Screenshot 2020-04-30 at 21 48 28](https://user-images.githubusercontent.com/2045367/80752771-6656ae00-8b2c-11ea-8897-ecc3b4211ce0.png)

**Comment by kgn** [comment]
Is this still not supported? Trying to import a DNS Domain Name to ENS and this requires DNSSEC.
https://medium.com/the-ethereum-name-service/step-by-step-guide-to-importing-a-dns-domain-name-to-ens-d2d15feb03e8

**Reply by sourabhxyz** [reply]
There is no need to follow this route. You can use your default name server (in my case, it's Google) and still use Vercel by adding records (A / CNAME) as explained [here](https://vercel.com/docs/concepts/projects/custom-domains) _in place_ of custom name servers. In addition to records required for Vercel deployment, also add your "_ens" record as shown in the article you shared.
